### PR TITLE
Ui fixes

### PR DESCRIPTION
**Pull request by mzayem**
Hi,

I added some of the ui changes and fixes. and make sure to make it responsive for mobile as well

regards, 

[video link here](https://drive.google.com/file/d/1SiVP1BPQlJwbSPhMDMcinauC-2EgpeDw/view?usp=sharing)

### CODE REVIEW

File: components/slides/HomePage.js [hunk 1]
```diff
@@ -56,24 +56,29 @@ export default function HomePage() {
...
-                    <RIGHT_ARROW_SVG />
-                  </button>
+                  <div className="mt-8 flex flex-col items-center sm:flex-row md:my-8 2xl:my-10">
+                    <input
+                      type="text"
+                      value={input}
+                      required={true}
+                      onChange={(e) => setInput(e.target.value)}
+                      autoFocus={true}
+                      className="focus:border-border-[#ECA227] inline w-full border-b-2 border-[#ECA227] bg-transparent p-2 py-3 text-lg outline-none transition-all duration-75 focus:rounded-full focus:border-2 focus:px-4 sm:w-11/12 sm:text-3xl md:w-10/12 md:text-xl lg:w-8/12 lg:text-xl 2xl:text-3xl"
```
Comment (itexpert120) [inline components/slides/HomePage.js]: Making this rounded on focus shifts the placeholder text. Which looks bad. Either keep it rounded all the time or try something else.

File: components/slides/HomePage.js [hunk 2]
```diff
@@ -56,24 +56,29 @@ export default function HomePage() {
...
+                      placeholder="Enter Your GitHub Username"
+                    />
+                    <button
+                      type="Submit"
+                      className="mx-2 mt-5 rounded-full border-2 border-white px-1 py-3 text-lg md:mt-0"
+                    >
+                      <RIGHT_ARROW_SVG className="m-0" />
```
Comment (itexpert120) [inline components/slides/HomePage.js]: On hover, the button size increases due to shift in arrow position

File: components/slides/Preview.js [hunk 3]
```diff
@@ -67,22 +67,18 @@ export default function Preview({ back }) {
   return (
     <div className="flex w-full flex-col items-center">
       <button
-        className="absolute left-0 m-10 opacity-80 outline-none transition-all ease-in-out hover:opacity-100"
+        className="absolute left-0 m-5 opacity-80 outline-none transition-all ease-in-out hover:opacity-100 md:m-10"
         onClick={back}
       >
         ◄ Go Back
       </button>
```
Comment (itexpert120) [inline components/slides/Preview.js]: Make this back button a re-useable component. and make it float in top left corner instead.

File: components/slides/TechStack.js [hunk 4]
```diff
@@ -99,7 +100,7 @@ ${profileMaker.data.tech
               name=""
               id=""
               value={searchStr}
-              className="max-w-[92vw] rounded-full bg-transparent p-4 px-8 pr-16 text-orange-300 outline-none ring-2 ring-orange-200 sm:max-w-full"
+              className="duration-400 max-w-[92vw] rounded-full bg-transparent p-3 px-6 pr-16 text-orange-300 outline-none ring-2 ring-orange-200 transition-all focus:ring-orange-500 sm:max-w-full md:p-4 md:px-8"
```
Comment (itexpert120) [inline components/slides/TechStack.js]: Keep text color white instead of orange to keep it consistent.

**Review by itexpert120 — CHANGES_REQUESTED** [review]

**Comment by itexpert120** [comment]
You should also fix:
- Add a container-screen-2xl class to the main container so the UI doesn't extend to screen edges.
- The UI breaks on mobile container sometime because the hat is too big
image
- The size of title on home screen should be responsive to screen
- Make the draggable zone on profile and background image selection more visible
- Make the generate using AI button opacity 50% while it is disabled
- Add a button to paste the "Tell us about yourself" placeholder into the textarea
- The select menu in github stats area is blending with background
- Use a globe icon instead of link icon in website url
- Make the tags section expandable and keep them contracted by default so it doesn't look too overwhelming
- Move theme for badges to the top
- Remove "Hey👋, Can you help us to grow by sharing?" for now
- Add required tag (small red *) to each input field that is required
- There is too much spacing between the hero content and the footer.

File: components/slides/HomePage.js (hunk 2 above)
Comment (mzayem) [reply components/slides/HomePage.js]: so isn't it look cool?

**Comment by mzayem** [comment]
All changes done 


> * Add required tag (small red *) to each input field that is required
there is only one required tag which is github username but those have no label so where we can add red mark
> * Use a globe icon instead of link icon in website url
i tried many globe i icons but they are not creating issues due to viewbox of our current svg reusable components only this link icon works and fits well but i am actively searching if can find a path for globe icon

**Review by mzayem — COMMENTED** [review]
Please reiview changes

File: components/slides/HomePage.js (hunk 2 above)
Comment (itexpert120) [reply components/slides/HomePage.js]: No, the button shape should not change

**Comment by itexpert120** [comment]
image
Fix this

**Comment by itexpert120** [comment]
image
shouldn't these too follow the same pattern as name input field?
image

**Comment by itexpert120** [comment]
image
This button still doesn't hover there while user scrolls down

**Comment by mzayem** [comment]
changes are done please review changes and merge this

**Review by itexpert120 — CHANGES_REQUESTED** [review]
Remove tags style selector
add ability to change uploaded picture

**Review by itexpert120 — APPROVED** [review]

**Comment by itexpert120** [comment]
LGTM

**Pull request merged**